[9.0][FIX] purchase_batch_invoicing: No Tax compute in invoice created

igallart opened the pull request.

pedrobaeza commented:
Please change module version number according https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

igallart committed:
[9.0][FIX] purchase_batch_invoicing: No tax compute when invoice is created

rafaelbn commented:
Dear @igallart please could you review travis in order to merge this?

rafaelbn reviewed (approved):
@igallart :+1:

pedrobaeza commented on purchase_batch_invoicing/wizards/purchase_batch_invoicing.py:
```diff
@@ -92,6 +92,7 @@ def action_batch_invoice(self):
                 invoice.currency_id = po.currency_id
                 invoice.purchase_id = po
                 invoice.purchase_order_change()
+                invoice._onchange_invoice_line_ids()
```
Why the onchange? This should be `invoice.compute_taxes` IMO

igallart replied:
If you think it's better to change it, that's fine with me. I can not prove it, but with the current solution it works well.

pedrobaeza replied:
Yes, because this might done more things than the expected one, depending also on the extension modules. See the patch we have made for v11: #505

rafaelbn replied:
How to suggest the change in the code? It saw now is possible in Github

rafaelbn replied:
```suggestion
                invoices.compute_taxes()
```

rafaelbn replied:
This is BETA TEST

igallart replied:
Changed.

pedrobaeza commented:
Why are you changing here things of account_invoice_merge?

igallart commented:
> Why are you changing here things of account_invoice_merge?

I have changed it based on the suggestions. I changed the position because I thought it was more correct to place it after the raise

pedrobaeza commented:
But this PR is for `purchase_batch_invoicing`. This has nothing to do with `account_invoice_merge`, and I don't see any comment about changing that here.

igallart commented:
You're right Pedro, it's been my fault. This happened because during the pull request I made two corrections in two modules, and github puts it on the same fork. I do not know if I can have two forks from the same repository at the same time. If it is very problematic, I will eliminate the extra one.

pedrobaeza commented:
You can't have more than one fork, but you are able to create the number of branches you need. I see that you have already fixed it and create another branch, so I think you have understood the concept.
